alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -129,9 +130,12 @@ protected function initLiveStreamConfig()
 		usort($liveEntryServerNodes, function ($a, $b) {return $b->weight - $a->weight;});
 		
 		$liveEntryServerNode = array_shift($liveEntryServerNodes); // after sort first is the primary
+		// If min/max bitrate was requested, add the constraint to the array of flavorParamsIds in the profile's attributes.
+		$this->retrieveStreamIdsByBitrate($liveEntryServerNode->getStreams());
```

Comment:
Primary and backup could have different streams. Although this usually does not happen.
The retrieveStreamIdsByBitrate does not handle this scenario.
Also, I think it's better to move this logic inside the liveEntryServerNode.